src/modules/networks/helpers.ts

```diff
@@ -1,20 +1,29 @@
-import { nanoid } from 'nanoid';
 import { invariant } from 'src/shared/invariant';
+import { normalizeChainId } from 'src/shared/normalizeChainId';
 import type { AddEthereumChainParameter } from '../ethereum/types/AddEthereumChainParameter';
+import { getCustomNetworkId } from '../ethereum/chains/helpers';
 import type { NetworkConfig } from './NetworkConfig';
-import type { Chain } from './Chain';
+
+export function getChainId(network: NetworkConfig) {
+  return network.standard === 'eip155'
+    ? normalizeChainId(network.specification.eip155.id)
+    : null;
+}
```

Comment:
What do you think about making this a static method of `Networks`?

Reply:
Ok, there is one moment I can't solve for now. 
There will be 2 `getChainId` methods inside Networks.
One is static accepting NetworksConfig, and one is not static, accepting Chain.
Is it ok to keep 2 methods with the same name?